CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Bug fixes
     + All other uses of `xarray`'s internal API were also removed (:pull:`2116`).
 * Fixed an issue with star-annotated call signatures to maintain Python 3.10 compatibility. (:pull:`2116`).
 * Fixed `to_agg_units` that was converting units of temperature differences prematurely, without changing accordingly the values in the related DataArrays. (:issue:`2121`, :pull:`2122`).
+* ``get_calendar`` now supports pandas' ``DatetimeIndex``. xclim no longer uses ``xarray.cftime_range``, which has been deprecated. (:pull:`2130`).
```

Comment:
```suggestion
* ``get_calendar`` now supports pandas' ``DatetimeIndex``. `xclim` no longer uses ``xarray.cftime_range``, which has been deprecated. (:pull:`2130`).
```

Do you know wince when it has been deprecated? Do we need a lower pin on `xarray` for this?

Reply:
We don't need a pin ! `date_range` has been there for some time, the solution should  be valid for all supported xarray.

The deprecation was announced in 2025.03. There are no "terminal" version indicated in the warning, nor in the PR.